<HTML>
<HEAD>
  <!-- Created by AOLpress/1.2 -->
  <TITLE>Mendelian Genetics</TITLE>
</HEAD>
<BODY BACKGROUND= "../gif/genebg.jpg" LINK="#0000ff" VLINK="#0000ff">
<P>
<TABLE BORDER=0  CELLPADDING=4>
  <TR>
    <TD COLSPAN=2>
      <IMG SRC="mendel.gif" WIDTH=225 HEIGHT=31 BORDER=0>    </TD>
  </TR>
  <TR>
    <TD VALIGN=TOP WIDTH=120>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE"><A HREF="mendel1.htm"> <FONT SIZE=-1>Mendel's First Law</FONT></A>
      <P>
      <FONT SIZE=-1><A HREF="mendel2.htm">Variations to Mendel's First Law</A></FONT>
      <P>
      <FONT SIZE=-1><A HREF="mendel9.htm">Pedigree Analysis</A></FONT>
      <P>
      <FONT SIZE=-1><A HREF="mendel3.htm">Mendel's Second Law</FONT></A>
      <P>
      <FONT SIZE=-1><A HREF="mendel4.htm">Chi-Square Test</FONT></A>
      <P>
      <FONT SIZE=-1><A HREF="mendel5.htm">Pleiotropy</FONT></A>
      <P>
      <FONT SIZE=-1><B>Epistasis</B></FONT>
      <P>
      <FONT SIZE=-1><A HREF="mendel7.htm">Modifier Genes</FONT></A>
      <P>
      <FONT SIZE=-1><A HREF="mendel8.htm">Penetrance and Expressivity</FONT></A>
      <P>
      <FONT SIZE=-1><A HREF="mendel_study.htm">Study Questions</FONT></A>
      <P>
      <FONT SIZE=-1><A HREF="../overheads/mendel/mend1.htm">Mendelian Genetics Overheads</FONT></A>
      <P>      
      <FONT SIZE=-1><A HREF="mendellinks.htm">Mendelian Genetics WWW Links</FONT></A>
      <P> 
      <FONT SIZE=-1><A HREF="../431g.htm">Genetic Topics</FONT></A>
      </FONT>     
    </TD>
    <TD WIDTH=415><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">
<H2>
  Gene Interactions
</H2>
<P>
 The genes of an individual do not operate isolated from
one another, but obviously are functioning in a common cellular environment.
Thus, it is expected interactions between genes would occur. Bateson and
Punnett performed a classical experiment that demonstrated genetic interactions.
They analyzed the three comb types of chicken known to exist at that time:
<P>
<TABLE BORDER CELLPADDING="2">
 <TR>
  <TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Chicken Varieties</FONT></TH>
  <TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Phenotype</FONT</TH>
 </TR>
 <TR>
  <TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Wyandotte</FONT></TD>
  <TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Rose Comb</FONT></TD>
 </TR>
 <TR>
  <TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Brahmas</FONT></TD>
  <TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Pea Comb</FONT></TD>
 </TR>
 <TR>
  <TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Leghorns</FONT></TD>
  <TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Single Comb</FONT></TD>
</TR>
</TABLE>
<P>   

<B>Rose</B>  <IMG SRC="chick1.gif" WIDTH=303 HEIGHT=142 BORDER=0><B>Pea</B><P>
<B>Single</B> <IMG SRC="chick2.gif" WIDTH=313 HEIGHT=144 BORDER=0><B>Walnut</B>

<P>
<IMG SRC="2-fig15a.gif" WIDTH=326 HEIGHT=88 BORDER=0>
<P>
 Result: The F<SUB>1</SUB> differed from both parents and two new phenotypes
not seen in the parents appeared in the F<SUB>2</SUB>. How can this result be explained?
The first clue is the F<SUB>2</SUB> ratio. We have seen this ratio before when the F<SUB>1</SUB>
from a dihybrid cross is selfed (or intermated). This observation suggests
that two genes may control the phenotype of the comb. The gene interactions
and genotypes were determined by performing the appropriate testcrosses.
<P>
A series of experiments demonstrated that the genotypes
controlling the various comb phenotypes are as follows.
<P>
<TABLE BORDER CELLPADDING="2">
  <TR>
    <TH>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Phenotypes</FONT>
    </TH>
    <TH>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Genotypes</FONT>
    </TH>
    <TH>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Frequency</FONT>
    </TH>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Walnut</FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE"><I>R_P_</I></FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">9/16</FONT>
    </TD>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Rose</FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE"><I>R_pp</I></FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">3/16</FONT>
    </TD>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Pea</FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE"><I>rrP_</I></FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">3/16</FONT>
    </TD>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Single</FONT>

    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE"><I>rrpp</I></FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">1/16</FONT>
    </TD>
  </TR>
</TABLE>

<P>
It was later shown that the genotypes of the initial parents
were:
<P>
<B>Rose</B> = <I>RRpp</I>
<P>
<B>Pea</B> = <I>rrPP</I>
<P>
Therefore, genotypically the cross was:
<P>
<IMG SRC="2-fig17a.gif" WIDTH=330 HEIGHT=197 BORDER=0>
<P>
 The development of any individual is obviously the expression
of all the genes that are a part of its genetic makeup. Therefore, it is
not an unexpected conclusion that more than one gene could be responsible
for the expression of a single phenotype. We will now discuss this situation.
First let's give a definition.
<P>
<STRONG>Epistasis</STRONG> - the interaction between two
or more genes to control a single phenotype
<P>
 The interactions of the two genes which control comb type
was revealed because we could identify and recognize the 9:3:3:1. Other genetic
interactions were identified because the results of crossing two dihybrids
produced a modified Mendelian ratio. All of the results are modifications
of the 9:3:3:1 ratio.
<P>
<STRONG>Example 1</STRONG>: 15:1 Ratio <BR>
<STRONG>Phenotypes:</STRONG> Kernel Color in Wheat
<P>
<IMG SRC="wheatkr2.gif" WIDTH=240 HEIGHT=103 BORDER=0>
<P>
<IMG SRC="2-fig18a.gif" WIDTH=267 HEIGHT=107 BORDER=0>
<P>
 For this type of pathway a functional enzyme A or B can
produce a product from a common precursor. The product gives color to the
wheat kernel. Therefore, only one dominant allele at either of the two loci
is required to generate the product.
<P>
 Thus, if a pure line wheat plant with a colored kernel (genotype
= <I>AABB</I>) is crossed to plant with white kernels (genotype = <I>aabb</I>) and the
resulting F<SUB>1</SUB> plants are selfed, a modification of the dihybrid 9:3:3:1 ratio
will be produced. The following table provides a biochemical explanation
for the 15:1 ratio.
<P>
<TABLE BORDER CELLPADDING="2">
  <TR>
    <TH>
       <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Genotype</FONT>
    </TH>
    <TH>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Kernel Phenotype</FONT>
    </TH>
    <TH>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Enzymatic Activities</FONT>
    </TH>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">9 <I>A_B_</I></FONT> 
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">colored kernels</FONT> 
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">functional enzymes from both genes</FONT> 
    </TD>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">3 <I>A_bb</I></FONT> 
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">colored kernels</FONT> 
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">functional enzyme from the <I>A</I> gene pair</FONT> 
    </TD>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">3 <I>aaB_</I></FONT> 
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">colored kernels</FONT> 
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">functional enzyme from the <I>B</I> gene pair</FONT> 
    </TD>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">1 <I>aabb</I></FONT> 
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">colorless kernels</FONT> 
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">non-functional enzymes produced at both genes</FONT> 
    </TD>
  </TR>
</TABLE>
<P>
<P>
If we sum the three different genotypes that will produce
a colored kernel we can see that we can achieve a 15:1 ratio. Because either
of the genes can provide the wild type phenotype, this interaction is called
duplicate gene action.
<P>
<STRONG>Example 2:</STRONG> 9:7 Ratio
<BR>
<STRONG>Example:</STRONG> Flower color in sweet pea
<P>
If two genes are involved in a specific pathway and functional
products from both are required for expression, then one recessive allelic
pair at either allelic pair would result in the mutant phenotype. This is
graphically shown in the following diagram.
<P>
<IMG SRC="2-fig20a.gif" WIDTH=348 HEIGHT=58 BORDER=0>
<P>
 If a pure line pea plant with colored flowers (genotype
= <I>CCPP</I>) is crossed to pure line, homozygous recessive plant with white flowers,
the F<SUB>1</SUB> plant will have colored flowers and a <I>CcPp</I> genotype. The normal ratio
from selfing dihybrid is 9:3:3:1, but epistatic interactions of the <I>C</I> and
<I>P</I> genes will give a modified 9:7 ratio. The following table describes the
interactions for each genotype and how the ratio occurs.
<P>

<TABLE BORDER CELLPADDING="2">
<TR>
<TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Genotype</FONT></TH>
<TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Flower Color</FONT></TH>
<TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Enzyme Activities</FONT>/TH>
</TR>
<TR VALIGN="Top">
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">9 <I>C_P_</I></FONT></TD>
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Flowers colored;<BR>anthocyanin produced</FONT></TD>
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Functional enzymes from both genes</FONT></TD>
</TR>
<TR VALIGN="Top">
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">3 <I>C_pp</I></FONT></TD>
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Flowers white;<BR>no anthocyanin produced</FONT></TD>
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">p enzyme non-functional</FONT></TD>
</TR>
<TR VALIGN="Top">
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">3 <I>ccP_</I></FONT></TD>
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Flowers white;<BR>no anthocyanin produced</FONT></TD>
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">c enzyme non-functional</FONT></TD>
</TR>
<TR VALIGN="Top">
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">1 <I>ccpp</I></FONT>  </TD>
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Flowers white;<BR>no anthocyanin produced</FONT></TD>
<TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">c and p enzymes non-functional</FONT></TD>
</TR>
</TABLE>

<P>
Because both genes are required for the correct phenotype,
this epistatic interaction is called <B>complementary gene action</B>.
<P>
<STRONG>Example 3:</STRONG> 12:3:1 Ratio
<BR>
<STRONG>Phenotype:</STRONG> Fruit Color in Squash
<P>
With this interaction, color is recessive to no color at
one allelic pair. This recessive allele must be expressed before the specific
color allele at a second locus is expressed. At the first gene white colored
squash is dominant to colored squash, and the gene symbols are <I>W</I>=white and
<I>w</I>=colored. At the second gene yellow is dominant to green, and the symbols
used are <I>G</I>=yellow, <I>g</I>=green. If the dihybrid is selfed, three phenotypes are
produced in a 12:3:1 ratio. The following table explains how this ratio is
obtained.
<P>
<IMG SRC="squash.gif" WIDTH=200 HEIGHT=143 BORDER=0>
<B>Shapes of Squash Fruit</B>
<P>

<TABLE BORDER CELLPADDING="2"><TR><TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Genotype</FONT>
</TH><TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Fruit Color</FONT>
</TH><TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Gene Actions</FONT>
</TH></TR><TR><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">9 <I>W_G_</I></FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">White</FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Dominant white allele negates effect of <I>G</I> allele</FONT>
</TD></TR><TR><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">3 <I>W_gg</I></FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">White</FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Dominant white allele negates effect of <I>G</I> allele</FONT>
</TD></TR><TR><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">3 <I>wwG_</I></FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Yellow</FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Recessive color allele allows yellow allele expression</FONT>
</TD></TR><TR><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE"><I>1 wwgg</I></FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Green</FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Recessive color allele allows green allele expression</FONT>
</TD></TR></TABLE>


<P>
Because the presence of the dominant <I>W</I> allele masks the effects
of either the <I>G</I> or <I>g</I> allele, this type of interaction is called
<STRONG>dominant epistasis</STRONG>.
<P>
<STRONG>Example 4:</STRONG> 13:3 ratio
<BR>
<STRONG>Phenotype:</STRONG> Malvidin production in <I>Primula</I>
<P>
Certain genes have the ability to suppress the expression
of a gene at a second locus. The production of the chemical malvidin in the
plant <I>Primula</I> is an example. Both the synthesis of the chemical (controlled
by the <I>K</I> gene) and the suppression of synthesis at the <I>K</I> gene (controlled
by the <I>D</I> gene) are dominant traits. The F<SUB>1</SUB> plant with the genotype <I>KkDd</I> will
not produce malvidin because of the presence of the dominant <I>D</I> allele. What
will be the distribution of the F<SUB>2</SUB> phenotypes after the F<SUB>1</SUB> was crossed?

<P>
<TABLE BORDER CELLPADDING="2">
  <TR>
    <TH>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Genotype</FONT>
    </TH>
    <TH>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Phenotype and genetic explanation</FONT>
    </TH>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">9 <I>K_D_</I></FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">no malvidin because dominant <I>D</I> allele is present</FONT>
    </TD>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">3 <I>K_dd</I></FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">malvidin productions because dominant <I>K</I> allele present</FONT>
    </TD>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">3 <I>kkD_</I></FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">no malvidin because recessive <I>k</I> and dominant D alleles
      present</FONT>
    </TD>
  </TR>
  <TR>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">1 <I>kkdd</I></FONT>
    </TD>
    <TD>
      <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">no malvidin because recessive <I>k</I> allele present</FONT>
    </TD>
  </TR>
</TABLE>
<P>
<P>
The ratio from the above table is 13 no malvidin production
to 3 malvidin production. Because the action of the dominant <I>D</I> allele masks
the genes at the <I>K</I> locus, this interaction is termed <STRONG>dominant suppression
epistasis</STRONG>.
<P>
<STRONG>Suppressor</STRONG> - a genetic factor that prevents
the expression of alleles at a second locus; this is an example of epistatic
interaction
<P>
 Remember that epistasis is the interaction between different
genes. If one allele or allelic pair masks the expression of an allele at
the second gene, that allele or allelic pair is epistatic to the second gene.
Therefore, the following table summarizes the four epistatic interactions
discussed above.
<P>

<TABLE BORDER CELLPADDING="2"><TR><TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Example</FONT>
</TH><TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Allelic Interactions</FONT>
</TH><TH><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Type of Epistasis</FONT>
</TH></TR><TR VALIGN="Top"><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Wheat kernel color</FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE"><I>A</I> epistatic to <I>B</I>, <I>b</I><BR><I>B</I> epistatic to <I>A</I>, <I>a</I></FONT> 
</TD><TD> <FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Duplicate genes</FONT>
</TD></TR><TR VALIGN="Top"><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Sweet pea flower color</FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE"><I>cc</I> epistatic to <I>P</I>, <I>p</I><BR><I>pp</I> epistatic to <I>C</I>, <I>c</I></FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Complementary gene action</FONT>
</TD></TR><TR><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Squash Fruit Color</FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE"><I>W</I> epistatic to <I>G</I>, <I>g</I></FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Dominant epistasis</FONT>
</TD></TR><TR><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Primula malvidin production</FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE"><I>D</I> epistatic to <I>K</I>, <I>k</I></FONT>
</TD><TD><FONT FACE="ARIAL, HELVETICA, AVANT GARDE">Dominant suppression</FONT>
</TD></TR></TABLE><P>
<B>Copyright &#169; 2000. Phillip McClean</B><P> 
    </TD>
  </TR>
</TABLE></FONT>
<P>

</BODY></HTML>
